app/src/main/java/com/example/bookclub/SearchActivity.java

```diff
@@ -92,7 +102,7 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
 
         rvBooks.setAdapter(bookAdapter);
         rvBooks.setLayoutManager(new LinearLayoutManager(this));
-        fetchBooks("Oscar Wilde");
+        fetchBooks("Malcolm Gladwell");
```

Comment:
Can you remove this later on? Code committed to master should be deployable code.

Reply:
I added that line so the user could see some books on the search screen when he on there. I didn't want to display a blank screen to the user.

Reply:
I think it's fine if the screen is blank. Or you can also pull the last search's results there so it's more relevant to the user.